pkg/webhook/cluster/mutation/mutation.go

```diff
@@ -136,6 +136,12 @@ func (h *AdmissionHandler) applyDefaults(c *kubermaticv1.Cluster) {
 		}
 	}
 
+	if c.Spec.ClusterNetwork.IPVS != nil {
```

Comment:
what if it is == nil?

Reply:
It gets defaulted in the kube-proxy template.